docs/v2_proposal.rst

```diff
@@ -0,0 +1,39 @@
+=======================
+OCX-Library V2 Proposal
+=======================
+
+The intention of this library is to act as a wrapper around the underlying client libraries.
+A user will import all of the "python-[service]client" libraries they would like to use, in addition to ours. 
+Our library relies on the existance of the other client libraries in the python environment.
+
```

Comment:
First, I think we need to explain why we need this wrapper libraries and why is essential to OCX. And it could be located in the first paragraph of this proposal.